### ISSUE TITLE
Error running ffmpeg, output is: on Arch Linux

### ISSUE THREAD

**Issue by dragonnn** [Niek/superview#14]
```
ffmpeg version: n4.3.1
Hardware accelerators: vdpau,cuda,vaapi,qsv,drm
H.264 encoders: libx264,libx264rgb,h264_nvenc,h264_omx,h264_qsv,h264_v4l2m2m,h264_vaapi
H.265/HEVC encoders: libx265,hevc_nvenc,hevc_qsv,hevc_v4l2m2m,hevc_vaapi
Scaling input file /home/mateusz/Dron/videos/RC_0056_2007292052.MP4 (codec: h264, duration: 326 secs) from 3840*2160 to 3840*2160 using superview scaling. Squeeze: true
Filter files generated
Re-encoding video with h264_qsv encoder at 57 MB/s bitrate
2020/08/01 22:00:44 Error running ffmpeg, output is:
&{%!s(*os.file=&{{{1 0 0} -1 {0} <nil> 0 0 true true true} |0 <nil> true false false})}
```

Can you help with that?
I just instaled and run the CLI version with a video from Runcam 5, recorded at 4K(XV) resolution.

**Comment by FinalFrag** [comment]
Having a similar issue when running the script in a docker container for a dji clip

```
ffmpeg version: 4.1.6-1~deb10u1
Hardware accelerators: vdpau,vaapi,drm
H.264 encoders: libx264,libx264rgb,h264_omx,h264_v4l2m2m,h264_vaapi
H.265/HEVC encoders: libx265,hevc_v4l2m2m,hevc_vaapi
Scaling input file /convert/output/DJIG0001_stabilized.mp4 (codec: h264, duration: 19 secs) from 960*720 to 1280*720 using superview scaling. Squeeze: false
Filter files generated
Re-encoding video with h264_vaapi encoder at 29 MB/s bitrate
2020/08/07 21:14:16 Error running ffmpeg, output is:
&{%!s(*os.file=&{{{1 0 0} -1 {0} <nil> 0 0 true true true} |0 <nil> true false false})}

**Comment by Niek** [comment]
Thanks for the reports! This took me a bit longer than expected to fix, but it should be resolved as of https://github.com/Niek/superview/commit/beeddb928a20628b77f321c92391df7563cad4c8

Can you please check again and report @dragonnn @FinalFrag?

There's also a prerelease of v0.7 (including the GUI version), see: https://github.com/Niek/superview/releases/tag/v0.7

**Comment by Niek** [comment]
Closing this, I assume it's resolved in v0.7 (released now).
